html/pfappserver/lib/pfappserver/Form/Config/Mfa/TOTP.pm

```diff
@@ -0,0 +1,90 @@
+package pfappserver::Form::Config::Mfa::TOTP;
+
+=head1 NAME
+
+pfappserver::Form::Config::Mfa::TOTP - Web form to add a TOTP MFA
+
+=head1 DESCRIPTION
+
+Form definition to create or update a TOTP MFA.
+
+=cut
+
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Mfa';
+with 'pfappserver::Base::Form::Role::Help';
+
+use pf::config;
+use pf::util;
+use File::Find qw(find);
+
+has_field 'scope' =>
+   (
+    type => 'Hidden',
+    default => 'Radius,Portal',
+   );
+
+has_field 'type' =>
+  (
+   typ => 'Hidden',
+   default => 'TOTP',
+  );
+
+has_field 'radius_mfa_method' =>
+  (
+   type => 'Select',
+   label => 'type',
+   required => 1,
+   options =>
+   [
+    { value => 'strip-otp', label => 'Strip OTP' },
+    { value => 'second-password', label => 'Second Password Field' },
+   ],
+   default => 'strip-otp',
+   tags => { after_element => \&help,
+             help => 'RADIUS MFA method' },
+  );
+
+has_field 'split_char' =>
```

Comment:
Shouldn't this field be in `pfappserver::Form::Config::Mfa ` since its common to all MFA modules?